dev/com.ibm.ws.jmx.connector.server.rest/src/com/ibm/ws/jmx/connector/server/rest/handlers/MBeanAttributeRouterHandler.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
```

Comment:
The objectName is not stored for other methods like https://github.com/OpenLiberty/open-liberty/blob/e9ab3f3e1b5a82af90b47b4730ca3e9b5d7f1280/dev/com.ibm.ws.jmx.connector.server.rest/src/com/ibm/ws/jmx/connector/server/rest/handlers/MBeanAttributeRouterHandler.java#L144

Just want to make sure that they don't need any changes.

Reply:
I couldn't see any reason to make any changes here. There don't seem to be any side effects of calling getRequiredParam. My guess is that the call is there to validate that the parameter exists, rather than to actually retrieve the value, and this will continue to work under servlet 6